crates/exec-wasmtime/Cargo.toml

```diff
@@ -9,7 +9,7 @@ license = "Apache-2.0"
 
 [dependencies]
 anyhow = { version = "1.0", default-features = false }
-cap-std = { version = "0.24.2", default-features = false }
+cap-std = { version = "=0.24", default-features = false }
```

Comment:
Leave a comment as to why we're locking this version?

Reply:
done